app/decorators/course_decorator.rb

```diff
@@ -69,6 +69,16 @@ def other_course_length?
     %w[OneYear TwoYears].exclude?(course.course_length)
   end
 
+  def other_age_range?
+    %w[3_to_7
+       5_to_11
+       7_to_11
+       7_to_14
```

Comment:
Can this list come from the edit_options?
eg If someone has a primary course but enters "11_to_18" as a custom age range (this might be valid for some SEND courses), we want it to show Other – this logic would give a false negative.

Reply:
Good spot. Done